stemcell_builder/stages/system_open_vm_tools/apply.sh

```diff
@@ -6,14 +6,59 @@ base_dir=$(readlink -nf $(dirname $0)/../..)
 source $base_dir/lib/prelude_apply.bash
 source $base_dir/lib/prelude_bosh.bash
 
-# Installation on CentOS requires v7
-pkg_mgr install open-vm-tools
+function compile_package {
+  run_in_chroot $chroot "
+    cd /tmp
+    apt install -y autoconf \
```

Comment:
Consider using `apt-get` instead of `apt`. The `apt` command line utility regularly warns that its interface is not stable and that it should not be relied on for scripting.

We don't appear to care about the output of the `apt` command in this script, though if we did in the future it could be an issue.

Reply:
https://askubuntu.com/questions/445384/what-is-the-difference-between-apt-and-apt-get
> The `apt` command is meant to be pleasant for end users and does not need to be backward compatible like apt-get(8).